its/org.sonarlint.eclipse.its.shared/src/org/sonarlint/eclipse/its/shared/reddeer/dialogs/ConfirmConnectionCreationDialog.java

```diff
@@ -22,12 +22,17 @@
 import org.eclipse.reddeer.swt.impl.shell.DefaultShell;
 
 public class ConfirmConnectionCreationDialog extends DefaultShell {
-  public ConfirmConnectionCreationDialog() {
-    super("Do you trust this SonarQube server?");
+  private final boolean isSonarCloud;
+
+  public ConfirmConnectionCreationDialog(boolean isSonarCloud) {
+    super(isSonarCloud
+      ? "Do you trust SonarCloud?"
```

Comment:
"Do you trust this SonarCloud organization?

Reply:
Changed, yes this is clearer